package.json

```diff
@@ -4,18 +4,22 @@
   "private": true,
   "dependencies": {
     "@giscus/react": "^2.0.3",
+    "@nhost/react": "^0.9.0",
     "@types/lodash": "^4.14.182",
     "@types/react": "^18.0.6",
     "@types/react-dom": "^18.0.2",
     "date-fns": "^2.28.0",
     "firebase": "^9.8.2",
+    "graphql": "^16.5.0",
```

Comment:
The `graphql` package is no more require.. right?

Reply:
Not required, @Angryman18 cannyou please remove it?

Reply:
removed